src/templates/fr/2019/contributors.html

```diff
@@ -0,0 +1,437 @@
+  }
+}
+
+@keyframes fade-in {
+  0% {
+    transform: translateY(30px);
+    opacity: 0;
+  }
+  30% {
+    opacity: 0;
+  }
+  100% {
+    transform: translateY(0);
+    opacity: 1;
+    }
+}
+</style>
+{% endblock %}
+
+{% block main %}
+<section class="main">
+  <h1 class="title title-lg">
+    Contributeurs et contributrices
+  </h1>
+  <div class="team">
+      <img src="/static/images/avatars/1.jpg" height="100" width="100" loading="lazy" />
+      <img src="/static/images/avatars/8.jpg" height="100" width="100" loading="lazy" />
+      <img src="/static/images/avatars/4.jpg" height="100" width="100" loading="lazy" />
+      <img src="/static/images/avatars/0.jpg" height="100" width="100" loading="lazy" />
+      <img src="/static/images/avatars/11.jpg" height="100" width="100" loading="lazy" />
+      <img src="/static/images/avatars/9.jpg" height="100" width="100" loading="lazy" />
+  </div>
+  <div class="contributors {{ ' '.join(config.teams.keys()) }}">
+    <div class="contributors-count"><span id="filtered-contributors">{{ config.contributors.items() | length }}</span> personnes</div>
+    <div class="filter-container">
+      <p>Filtrer par équipe&nbsp;:</p>
+      <button class="analysts" type="button">Analyse</button>
+      <button class="authors" type="button">Rédaction</button>
+      <button class="brainstormers" type="button">Réflexion</button>
+      <button class="designers" type="button">Design</button>
+      <button class="developers" type="button">Développement</button>
+      <button class="editors" type="button">Édition</button>
+      <button class="reviewers" type="button">Relecture</button>
+      <button class="translators" type="button">Traduction</button>
+    </div>
+
+    <div class="contributor-grid">
+    {% for id, contributor in config.contributors.items() %}
+      <div class="contributor {{ ' '.join(contributor.teams) }}" id="{{ id }}" tabindex="-1">
+        <img class="contributor-avatar" src="{{contributor.avatar_url}}" height="100" width="100" alt="{{ contributor.name }} avatar" loading="lazy" />
+        <div class="contributor-name">{{ contributor.name }}</div>
+
+        <div class="contributor-social">
+          {% if contributor.github %}
+          <a href="https://github.com/{{ contributor.github }}" target="_blank" title="{{ contributor.github }} on GitHub"><img src="/static/images/github.png" alt="GitHub" height="80" width="78" loading="lazy" /></a>
+          {% endif %}
+          {% if contributor.twitter %}
+          <a href="https://twitter.com/{{ contributor.twitter }}" target="_blank" title="{{ contributor.twitter }} on Twitter"><img src="/static/images/twitter.png" alt="Twitter" height="80" width="65" loading="lazy" /></a>
+          {% endif %}
+          {% if contributor.website %}
+          <a href="{{ contributor.website }}" target="_blank" title="{{ contributor.name }} website"><img src="/static/images/blog.png" alt="Site Web" height="72" width="72" loading="lazy" /></a>
```

Comment:
website? Or site web?

```suggestion
          <a href="{{ contributor.website }}" target="_blank" title="{{ contributor.name }} site web"><img src="/static/images/blog.png" alt="Site Web" height="72" width="72" loading="lazy" /></a>
```